homework_1/zad1/solution/testCases_test.go

```diff
@@ -0,0 +1,34 @@
+package solution_test
+
+type testCase struct {
+	inputStart int
+	inputEnd int
+
+	expectedOutput []string
+	expectingError bool
+}
+
+func getTestCases() []testCase {
```

Comment:
I'd add few more edge cases (e.g. `(0, 0)`, `(-1, -5)`, `(-5, -1)`, `(-5, 5)`).
Btw. you didn't handle negative numbers as unexpected values, which I suppose is fine (game rules don't really say anything regarding negative values). Not sure if you've done it intentionally or not, but I just wanted to point out it is very important to have appropriate test cases if you're supporting negative values and 0 value.